CS410_Assignment2_CodeReview/src/sensormain/SensorApplication.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Lauren Kearney
+ * CS 410
+ * 
+ * three panel GUI for Sensor application
+ * 
+ * */
+
+package sensormain;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.util.ArrayList;
+import java.util.List;
+
+import java.awt.*;
+import javax.swing.*;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JProgressBar;
+import javax.swing.border.TitledBorder;
+
+import sensor.PressureSensor;
+import sensor.RadiationSensor;
+import sensor.TemperatureSensor;
+
+public class SensorApplication extends JFrame {
+	
+	public PressureSensorAdapter pressure = new PressureSensorAdapter(new PressureSensor());
+	public RadiationSensorAdapter radiation = new RadiationSensorAdapter(new RadiationSensor());
+	public TemperatureSensorAdapter temperature = new TemperatureSensorAdapter(new TemperatureSensor());
+	
+	public SensorApplication() {
+		
+		int pressureLength = (int) (pressure.value());
+		pressureLength *= 10;
+		int radiationLength = (int) (radiation.value());
+		radiationLength *= 10;
+		int temperatureLength = (int) (temperature.value());
+		temperatureLength /= 4;
+		

```

Comment:
now you can use the same values throughout the gui
ex: `int pressureLength = (int) pressureValue`